packages/integration-sdk-core/src/types/step.ts

```diff
@@ -119,6 +120,10 @@ export interface StepRelationshipMetadata extends StepGraphObjectMetadata {
   _class: RelationshipClass;
   sourceType: string;
   targetType: string;
+  /**
+   * A date for when there will be no guarantee that the relationship will continue to be ingested.
+   */
+  deprecationDate?: DateYMDString;
```

Comment:
Since this is on both `StepEntityMetadata` and `StepRelationshipMetadata`, it'd be great if this property lived on `StepGraphObjectMetadata`. We can then deprecate `StepMappedRelationshipMetadata` in the same fashion.

Reply:
I want sure if we wanted to use this process for deprecating Entities as well, but I'm good with putting it into StepGraphObjectMetadata